examples/a_scan_input_file_IN.DAT

```diff
@@ -239,7 +239,7 @@ boundl(29) = 0.1
 bore = 2.0
 
 * fvsbrnni
-ixc = 44
+* ixc = 44
```

Comment:
Ignore that, error 243 is a Level 2 error that has always happened. Its just being treated as Level 3 because of #3019. If you remove this I will get a fix into main ASAP.